lib/internal/Magento/Framework/Xml/Test/Unit/_files/data.xml

```diff
@@ -10,6 +10,7 @@
     <text> some text </text>
     <trim_spaces>   </trim_spaces>
     <cdata><![CDATA[  Some data here <strong>html</strong> tags are <i>allowed</i>  ]]></cdata>
+    <multiple_cdata>  <![CDATA[One]]>  <![CDATA[Two]]>  <![CDATA[Three]]>  </multiple_cdata>
```

Comment:
Hi @fnogatz,
Could you use more advanced test date here? Something like this:
```suggestion
    <multiple_cdata>  <![CDATA[Some data]]> here<![CDATA[ <strong>html</strong> tags]]> are <![CDATA[<i>allowed</i>]]> </multiple_cdata>
```

Reply:
Good suggestion. It actually helped to identify another problem for the case CDATA and text nodes are mixed. Resolved via 71e0068 and 683c2e2.